ray_ci/pipeline_ci.py

```diff
@@ -294,7 +303,13 @@ def _print_command(cmd: str) -> List[str]:
         pipeline_steps = inject_commands(pipeline_steps, before=setup_commands)
 
     # Print to stdout
-    steps_str = json.dumps(pipeline_steps)
+    if group_name:
+        pipeline_steps = {
+            "group": group_name,
+            "steps": pipeline_steps,
+        }
+
+    steps_str = json.dumps({"steps:": pipeline_steps})
```

Comment:
based on buildkite official docs, steps should have a `steps` as key, rather than just a raw list.